src/main/resources/default/taglib/w/filefield.html.pasta

```diff
@@ -53,7 +55,7 @@
                     value = value.substr(0, value.lastIndexOf("/"))
                 }
 
-                selectVFSFile(value).then(function (selectedValue) {
+                selectVFSFile(value, null, '___allowedExtensions').then(function (selectedValue) {
```

Comment:
this is a bit messy, but I didn't want to restructure the wondergem methods, in tycho this is cleaner thanks to the config object